### ISSUE TITLE
Post giraffe-develop merge cleanup

### ISSUE THREAD

**Issue by mikevespi** [bcgov/cas-reporting#215]
#### Describe the task

In order to keep the branch used to merge Giraffe work into `develop` isolated, the comments made in [that PR](https://github.com/bcgov/cas-registration/pull/1268) are yet to be addressed. We need to make the necessary changes, and remove any references to the `giraffe-develop` that made it to the code (e.g. in git workflows). 

#### Acceptance Criteria

- [x] comments from [the giraffe-develop merge PR](https://github.com/bcgov/cas-registration/pull/1268) have been addressed
- [x] all necessary changes to the helm chart have been made
- [x] no longer relevant references to giraffe-develop have been removed or updated
- [x] check the makefile

#### Additional context

More changes that need to be made now that we've merged may emerge as this work is done. If they are small enough to include in this work, feel free to do so. Otherwise write a new ticket for it.

#### Blocked By
#232

**Comment by pbastia** [comment]
I wonder how much of this is left. We should definitely do it before a lot of code is built on top

**Comment by joshgamache** [comment]
**Soft-blocked** by #232. Since that ticket will be rewriting/refactoring a large block of CI, it might inherintly take care of pieces of this ticket. Review this ticket after completing 232.

**Comment by BallardRobinett** [comment]
Removing use nx affected, add tags, and refactor filename AC since they are included in this card: https://github.com/bcgov/cas-reporting/issues/232

**Comment by BallardRobinett** [comment]
to confirm, I am totally removing the deploy process to cif-tools that giraffe-dev was using? @pbastia @joshgamache

**Comment by pbastia** [comment]
Correct @BallardRobinett ! Our frontend pods now share the same API/database, so there is no point in a separate deployment anymore
